root/etc/e-smith/templates/httpd/proxy-extra/20ssl_engine10Default

```diff
@@ -0,0 +1,20 @@
+{
+    use JSON;
+    use File::Basename;
+    my $certs = decode_json(`/usr/libexec/nethserver/cert-list`);
+
+    if ($Port == 443) {
+        $OUT .= "    #20ssl_engine\n";
+        $OUT .= "    SSLEngine On\n";
+        $OUT .= "    SSLProxyEngine On\n";
```

Comment:
Should the SSLProxyEngine  be turned on only if target url scheme is `https`?

Reply:
yep we could remove it, let me check